Look for memfd_create in sys/mman.h instead of linux/memfd.h.
Needed to build with glibc-2.27.

--- a/configure.ac	1969-12-31 19:00:00.000000000 -0500
+++ b/configure.ac	2018-03-27 23:54:15.414589005 -0400
@@ -360,7 +360,7 @@
 # ------------------------------------------------------------------------------
 
 AC_CHECK_HEADERS([sys/capability.h], [], [AC_MSG_ERROR([*** POSIX caps headers not found])])
-AC_CHECK_HEADERS([linux/memfd.h], [], [])
+AC_CHECK_HEADERS([sys/mman.h], [], [])
 
 AC_CHECK_HEADERS([printf.h], [have_printf_h=yes], [have_printf_h=no])
 AS_IF([test x$have_printf_h = xyes], [
@@ -395,6 +395,7 @@
                 [], [], [[
 #include <sys/types.h>
 #include <unistd.h>
+#include <sys/mman.h>
 #include <sys/mount.h>
 #include <fcntl.h>
 #include <sched.h>
